openshift/Makefile

```
@@ -1,4 +1,17 @@
-AP := ansible-playbook -vv -c local -i localhost,
+AP := ansible-playbook -e ansible_python_interpreter=/usr/bin/python3.14 -vv -c local -i localhost,
 
 deploy:
 	$(AP) deploy.yml
+
+images:
```

checkmake (0.3.2)

[warning] Target body for "images" exceeds allowed length of 5 lines (11).

(maxbodylength)